build.gradle

```diff
@@ -28,6 +28,7 @@ buildscript {
         classpath group: 'gradle.plugin.com.github.spotbugs', name: 'spotbugs-gradle-plugin', version: spotbugsPluginVersion
         classpath "org.ajoberstar:grgit:${gradleGitPluginVersion}"
         classpath "io.franzbecker:gradle-lombok:${gradleLombokPluginVersion}"
+        classpath fileTree(dir: "$rootDir/libs", include: "*.jar")
```

Comment:
This file should not be checked in - https://github.com/ravibeta/pravega-storage-adapters-1/tree/issue-11-hdfs-move-1/libs
we should figure out a way to depend on published lib.

Reply:
Yes Ravi I did https://github.com/pravega/pravega/pull/5125